apps/next/src/popup/index.tsx

```diff
@@ -28,23 +29,45 @@ const App = lazy(() => {
 
 const root = createRoot(document.getElementById('popup') as HTMLElement);
 
+export const Providers = ({
+  providers,
+  children,
+}: {
+  providers: ReactNode[];
+  children: ReactElement;
+}) => {
+  const renderProvider = (renderedProviders, renderedChildren) => {
```

Comment:
Missing argument types
```suggestion
  const renderProvider = (renderedProviders: ReactElement[], renderedChildren: ReactElement) => {
```